src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java

```diff
@@ -527,6 +527,17 @@ private boolean checkUserAuthorised(String jwt, Map<String, Object> model) {
         return true;
     }
 
+    private String obfuscateEmailAddress(String email) {
+        int start = email.indexOf("@");
```

Comment:
Could use the same approach as in the api class InsightOperationReporter.disguiseEmail - that way you get the first three characters as well

Reply:
Yep, will change for consistency